pkg/engine/task/task_pipe_test.go

```diff
@@ -68,6 +68,8 @@ func Test_isRelative(t *testing.T) {
 	}
 
 	for i, test := range tests {
```

Comment:
Not necessarily part of this PR, but wdyt of normalizing this test?
```suggestion
	for _, tt := range tests {
	    ...
```

Reply:
I'd normalize to `test`, `tt` is non-descriptive ;)

Reply:
Stop fighting [Go conventions](https://github.com/golang/go/wiki/TableDrivenTests) 👅

Reply:
Someone using `tt` in a Wiki doesn't make this a convention 😉

Reply:
Still, [resisting](https://github.com/golang/go/wiki/CodeReviewComments#variable-names)?